core/src/main/java/edu/wpi/grip/core/util/Holder.java

```diff
@@ -0,0 +1,42 @@
+
+package edu.wpi.grip.core.util;
+
+/**
+ * Simple thread-safe class to allow for setting values from lambdas.
+ */
+public final class Holder<T> {
```

Comment:
Ah, it used to be before I refactored out the lambdas. I can just remove it, then.